libraries/lib-datahandler/src/main/thrift/projects.thrift

```diff
@@ -111,6 +118,8 @@ struct ProjectLink {
 //    5: optional string parentId,
     6: optional string nodeId,
     7: optional string parentNodeId,
+    8: optional ProjectType projectType,
+    9: optional ProjectClearingState clearingState,
```

Comment:
ProjectLinks in general are not persisted and not covered by moderation requests